tests/cypress/README.md

```diff
@@ -19,13 +19,18 @@
 
 ## Running tests
 
-To run cypress tests locally you need to pass in environment variables for the state user and admin user passwords.
+To run cypress tests locally you will go to the root of the project and you'll need to pass in environment variables for the state user and admin user passwords.
 The final command will look something like this:
 
 `CYPRESS_ADMIN_USER_PASSWORD=passwordhere CYPRESS_STATE_USER_PASSWORD=passwordhere yarn test`
 
 If you don't have these passwords you can find them in AWS SSM parameters in the mdct-mcr-dev account. Look for the parameter with a name like `/configuration/default/cognito/bootstrapUsers/password`. Ask a repository contributor for help if needed.
 
+If you run into errors after trying to run the cypress test command:
+ -try running `yarn` at the root of the project
+ -run `nvm use` along with the current version that the project is on
```

Comment:
Nit: Just running `nvm use` anywhere in the project will do the trick. When you run `nvm use` it traverses the directory structure upward looking for `.nvmrc` files. Once it finds one (in our case it's at the project root), it will use the value in that file as the version. 👍